doc/nrf/releases/release-notes-changelog.rst

```diff
@@ -359,6 +359,9 @@ Libraries for networking
 
   * Added support for using X509 certificates.
 
+* :ref:`lib_fota_download` library:
+
+  * Added error code :c:enumerator:`FOTA_DOWNLOAD_ERROR_CAUSE_INTERNAL` when initialization of the download fails and source of error is not in the network.
```

Comment:
```suggestion
  * Added an error code :c:enumerator:`FOTA_DOWNLOAD_ERROR_CAUSE_INTERNAL` to indicate the failure of initialization of download, where the source of error is not in the network.
```

Reply:
This changes the meaning a little bit as it might not always be the initialization phase that fails. Maybe I could concise the message like follows:

```
* Added an error code :c:enumerator:`FOTA_DOWNLOAD_ERROR_CAUSE_INTERNAL` to indicate that the source of error is not network related.
```